templates/telemetry/crash/crash.4.schema.json

```diff
@@ -162,6 +162,72 @@
         "processType": {
           "type": "string"
         },
+        "stackTraces": {
+          "type": ["object", "null"],
+          "properties": {
+            "crash_info": {
+              "type": "object",
+              "properties": {
+                "address": @TELEMETRY_HEXADDRESS_1_JSON@,
```

Comment:
Do we really want to throw out the ping if one of these hex fields isn't in the expected format? Should we just let these be strings without validation?

Reply:
On the fence about this -- the fact that all the pings we're seeing in the field are formatting this correctly make me think we should be strict about this so we notice if people start submitting garbage. The main point of this telemetry is to allow us to create a signature of crashes that are being submitted, I'd like to be notified directly if that seems to be impossible.